README.md

```diff
@@ -1,12 +1,22 @@
 # ChoralWeb
 
 ## Installation
+You need a instance of redis running easiest way to get it is with docker
+
+docker run -d --name redis -p 6379:6379 redis
+
+This starts up a instance on localhost:6379 which is the defaut for the app
+
+Now overwrite the config file with the correct clientID and clientSecret you will need to get them from google. The file needs to be saved under config/index.js
```

Comment:
Do you have an example of this? I think I get what you are saying but I want to be sure.